JenkinsfileRelease

```diff
@@ -0,0 +1,80 @@
+pipeline {
+
+  parameters {
+    gitParameter name: 'branch',
+                description: 'The branch to check out',
+                branchFilter: 'origin/(.*)',
+                defaultValue: 'master',
+                selectedValue: 'DEFAULT',
+                type: 'PT_BRANCH',
+                listSize: '0',
+                quickFilterEnabled: true
+
+    string name: 'ReleaseVersion',
+         description: 'Version to release',
+         defaultValue: ''
+  }
+
+  agent {
+    label 'graphdb-jenkins-node'
+  }
+
+  options {
+    disableConcurrentBuilds()
+    timeout(time: 15, unit: 'MINUTES')
```

Comment:
Isn't this too short?

Reply:
We are not running tests as part of the release, so it should be enough, do you have other concerns?

Reply:
alright then